src/BugsnagServiceProvider.php

```diff
@@ -269,8 +269,8 @@ protected function setupCallbacks(Client $client, Container $app, array $config)
         if (!isset($config['user']) || $config['user']) {
             $client->registerCallback(new CustomUser(function () use ($app) {
                 if ($user = $app->auth->user()) {
-                    if (is_callable([$user, 'toArray'])) {
-                        return $user->toArray();
+                    if (is_callable([$user, 'attributesToArray'])) {
```

Comment:
Hmm I see. Only models then? Would that be the case for the [logger](https://github.com/bugsnag/bugsnag-laravel/blob/130644ec4632fe48c3979a4f58501d9cb8e5e5c3/src/MultiLogger.php#L71) too? That's the only other place we're using `is_callable`.